hivemind/moe/server/runtime.py

```diff
@@ -108,6 +105,11 @@ def run(self):
                 if not self.shutdown_trigger.is_set():
                     self.shutdown()
 
+    def process_batch(self, pool: TaskPoolBase, batch_index: int, *batch: torch.Tensor) -> Tuple[Any, int]:
```

Comment:
`batch_index` is not used, is it okay?

Reply:
yep, it is meant to be exposed for subclassing